### PR TITLE
remove attachMediaStream usage

### PR DESCRIPTION
**Pull request by fippo**
second part of the change proposed in https://github.com/webrtc/samples/pull/646#issuecomment-142157106 -- removes attachMediaStream. And some comments related to it.

Even the odd comment in peerconnection/audio that claims attachMediaStream is used even though it is not.

### CODE REVIEW

File: src/content/peerconnection/constraints/js/main.js [hunk 1]
```diff
@@ -158,7 +158,7 @@ function createPeerConnection() {
   };
   remotePeerConnection.onaddstream = function(e) {
     console.log('remotePeerConnection got stream');
-    attachMediaStream(remoteVideo, e.stream);
+    remoteVideo = e.stream;
```
Comment (samdutton) [inline src/content/peerconnection/constraints/js/main.js]: This should be `remoteVideo.srcObject = e.stream` (shouldn't it?)

**Comment by samdutton** [comment]
LGTM apart from comment.

File: src/content/peerconnection/constraints/js/main.js (hunk 1 above)
Comment (fippo) [inline src/content/peerconnection/constraints/js/main.js]: good catch

**Comment by fippo** [comment]
looks like jshintrc needs a more general cleanup. Removed some obsolete stuff but there is still some things which I don't know about. Well... jshint still works.

**Comment by samdutton** [comment]
LGTM.

**Comment by samdutton** [comment]
Pushed to gh-pages — LGTM.

**Pull request merged**